bundles/org.openhab.binding.meross/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meross"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
```

Comment:
While not wrong, i personally prefer a different id. As the bridge is also a conceptual item in openHAB, it might be a little confusing for new users. Maybe gateway or similar is better?

Reply:
ok I  agree I will use gateway instead, I will change the bridge/gateway  java and xml constants accordingly